analytics-firebase/src/main/java/com/mindera/skeletoid/analytics/appenders/FbAppender.kt

```diff
@@ -61,7 +61,7 @@ class FbAppender : IAnalyticsAppender {
         firebaseAnalytics?.setUserId(userId)
     }
 
-    override fun setUserProperty(name: String, value: String) {
+    override fun setUserProperty(name: String, value: String?) {
         if (firebaseAnalytics == null) {
```

Comment:
🤔 What is the usage of sending a null value? Clean the previous value?

Reply:
yes, and it's allowed by firebase, the method signature was just wrong after I migrated it to Kotlin